packages/devextreme/js/ui/tabs.js

```diff
@@ -397,16 +397,12 @@ const Tabs = CollectionWidget.inherit({
         return scrollableDirection;
     },
 
-    _updateScrollableDirection() {
-        const scrollable = this.getScrollable();
-
-        if(scrollable) {
-            const scrollableDirection = this._getScrollableDirection();
-
-            scrollable.option('direction', scrollableDirection);
-        } else {
-            this._renderScrolling();
+    _updateScrollable() {
+        if(this.getScrollable()) {
+            this._cleanScrolling();
         }
+
+        this._renderScrolling();
```

Comment:
The solution seems to be very wide, cannot we just call `_updateNavButtonsVisibility` or localize it other way?

Reply:
I believe we can use `showNavButton` method (possibly with some modifications). In common, it's not a good practice to rerender so wide parts. But now I see that it's already written in dimension changed handlers, so your new code definitely does not the common performance worse.

So let's postpone it for now.